README.md

```diff
@@ -25,19 +25,38 @@ There is never too much static code analysis. Try it out.
 
 Latest documentation is available in [docs/overview.md](docs/overview.md).
 
-## Quick start / installation
+## Installation
 
 ```
 go get -u github.com/PieselBois/kfulint/...
 ```
 
+## Usage
+
 Add linter to your `$PATH` or run it directly:
 
 ```
 ./bin/kfulint package/path/to/lint
 ```
 
-Run `./bin/kfulint -help` for more info.
+Usage of **kfulint**: `kfulint [flags] [package]`
+
+Package: package name (one package for now)
+
+Flags:
+```
+  -help
```

Comment:
I would better write smth like 
**-enable [checker name]** -  comma-separated list of enabled checkers (default "all")
and so on